test/functional/plugin/shada_spec.lua

```diff
@@ -184,7 +154,7 @@ describe('In autoload/shada.vim', function()
         '  + b                 2',
         '  + c    column       3',
         '  + d                 4',
-      }, {{type=1, timestamp=0, data=st({a=1, b=2, c=3, d=4})}})
```

Comment:
AFAIR it is needed for the test because to check that resulting order does not depend on initial order one needs to have known initial order. Using different order will not make test fail if everything is fine, but it may make test *not* fail if something is *not* fine with the ordering. Though it looks like here test was not actually doing what intended: before data reaches tested VimL function it will be transformed into a VimL dictionary: without keeping order obviously.